### PR TITLE
Update FilterTypeGuesser.php

### PR DESCRIPTION
**Pull request by cassianotartari**
Typo under FilterTypeGuesser.php, wrong filter type service name. `TypeGuess` needs a filter service as first parameter.

### CODE REVIEW

File: Guesser/FilterTypeGuesser.php [hunk 1]
```diff
@@ -71,7 +71,7 @@ public function guessType($class, $property, ModelManagerInterface $modelManager
                 $options['field_type']    = 'sonata_type_boolean';
                 $options['field_options'] = array();
 
-                return new TypeGuess('sonata_search_elastic_boolean', $options, Guess::HIGH_CONFIDENCE);
+                return new TypeGuess('sonata_search_elastica_boolean', $options, Guess::HIGH_CONFIDENCE);
```
Comment (core23) [inline Guesser/FilterTypeGuesser.php:74]: Sure it is elasticA?

Comment (cassianotartari) [inline Guesser/FilterTypeGuesser.php:74]: Sure, you can check under services declaration, it's a filter type:

``` xml
        <service
            id="sonata.admin.search.filter.type.boolean"
            class="Sonata\AdminSearchBundle\Filter\BooleanFilter">
            <tag name="sonata.admin.filter.type" alias="sonata_search_elastica_boolean" />
        </service>
```

Comes from https://github.com/ruflin/Elastica

**Comment by core23** [comment]
Please update the commit message and PR title @cassianotartari

**Comment by cassianotartari** [comment]
Like this?

**Comment by greg0ire** [comment]
Not really no… I fixed it for you while squashing and merging, but you will probably find [this](https://github.com/Soullivaneuh/sonata-dev-kit/blob/doc-contributing/doc/CONTRIBUTING.md#the-commit-message) very interesting.

Anyway,  thanks for your contribution.

**Pull request merged**